plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/CloudToolsInfo.java

```diff
@@ -26,6 +26,8 @@
   // Don't change the value; this name is used as an originating "application" of usage metrics.
   public static String METRICS_NAME = "gcloud-eclipse-tools";
 
+  public static String ANALYTICS_USER_AGENT = METRICS_NAME + getToolsVersion();
```

Comment:
Just USER_AGENT would do. We'll probably want to use this name for non-analytics things too.